benchmark/interaction/chartjs-master.html

```diff
@@ -32,7 +32,7 @@
             },
             options: {
                 responsive: false,
-                devicePixelRatio: 1,
+                //devicePixelRatio: 1,
```

Comment:
I specify `devicePixelRatio` to ensure the same amount of pixels are rendered by each library for a fair comparison, and to keep the reproducibility of my benchmark cross different monitors. Any reason you don't like this?

Reply:
No reason, was just an test and not intentional in the pr.